aiidalab_widgets_base/utils/__init__.py

```diff
@@ -153,3 +154,12 @@ class StatusHTML(_StatusWidgetMixin, ipw.HTML):
     @traitlets.observe("message")
     def _observe_message(self, change):
         self.show_temporary_message(change["new"])
+
+
+def ase2spglib(ase_structure: Atoms):
+    """convert ase Atoms instance to spglib cell"""
+    lattice = ase_structure.get_cell()
+    positions = ase_structure.get_scaled_positions()
+    numbers = ase_structure.get_atomic_numbers()
+
+    return (lattice, positions, numbers)
```

Comment:
Is this the definition of an spglib cell? Can we maybe add a link to the corresponding docs to the doc-string?

Reply:
Link https://spglib.github.io/spglib/python-spglib.html#crystal-structure-cell added to docstring.